### DISCUSSION TITLE
Rollback to previous state [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2188] [Discord Threads]
> 🙋‍♂️ **vikiiingur**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### This is not yet available. This can be substantial effort given we connect to any database.  
#  
> 🙋‍♂️ **vikiiingur**  
#### which means?  
#  
> 💁 **o1lab**  
#### to rollback any database to some point in time / tag - correct me if I am not understand this right  
#  
> 💁 **o1lab**  
#### or is it just an individual change that you would like to rollback from previously made changes  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976731954902888478)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
